Created 'LWC' App Page with 'accountsComponent' lwc

AmigoBongero opened the pull request:
- created 'LWC' App Page and added it to Sales App
- created 'accountsComponent' lwc and developed html markdown
- created new standard user 
- created LWC Permission set 
- assigned Permission set to standard user 
- assigned permission set to Sales App 

Time spent: 3 hours

AmigoBongero committed:
#1 Created accountsComponent

- dev: developed html markdown of accounts and three included tabs

IJWMG commented on DevJourney/.idea/SF-Dev-Journey.iml:
@AmigoBongero Add .idea directory and .iml files to gitignore, it's your IDE setup config

IJWMG commented on DevJourney/DevJourney.iml:
@AmigoBongero Same, to .gitignore

IJWMG commented on DevJourney/force-app/main/default/applications/standard__Sales.app-meta.xml:
@AmigoBongero Is that the right application? Can't see difference from the standard app at all

IJWMG commented on DevJourney/force-app/main/default/tabs/LWC.tab-meta.xml:
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>Created by Lightning App Builder</description>
+    <flexiPage>LWC</flexiPage>
```
@AmigoBongero Add Flexipage to the repo please

IJWMG commented on DevJourney/force-app/main/default/lwc/accountsComponent/accountsComponent.html:
```diff
@@ -0,0 +1,16 @@
+                </lightning-tabset>
+            </lightning-tab>
+            <lightning-tab label="Test">
+
```
@AmigoBongero 11-13 lines - make it one line. You literally did it three times before :)

AmigoBongero committed:
#2 Fixes

- added right application
- added flexipage
- fixed accountsComponent.html
- changed gitignore file

AmigoBongero reviewed (commented):
#Fixes 
- added right application
- added flexipage
- fixed accountsComponent.html
- changed gitignore file

IJWMG commented on DevJourney/force-app/main/default/applications/standard__LightningSales.app-meta.xml:
@AmigoBongero It would be better to create a new one, but let it be

IJWMG commented on DevJourney/force-app/main/default/applications/standard__LightningSales.app-meta.xml:
```diff
@@ -0,0 +1,33 @@
+    <tabs>standard-Case</tabs>
+    <tabs>standard-Forecasting3</tabs>
+    <tabs>Case_Party__c</tabs>
+    <tabs>Expense__c</tabs>
```
@AmigoBongero What are these? Do you have them in your repo? Do we need them at all here?

IJWMG commented on DevJourney/force-app/main/default/applications/standard__Sales.app-meta.xml:
@AmigoBongero I bet this one is extra :)

IJWMG commented on DevJourney/.gitignore:
@AmigoBongero Try to google this problem

AmigoBongero committed:
# Remove idea directory and DevJourney.iml from tracking

AmigoBongero committed:
#Added husky and vscode directories to gitingore file

AmigoBongero committed:
Removed standard__Sales.app-meta.xml

AmigoBongero committed:
#Removed tabs from LightningSales app
